start.sh

```diff
@@ -24,6 +24,9 @@ if [ -z "${IS_CRON_POD}" ]; then
     # Start Gunicorn processes
     echo Starting Gunicorn.
 
+    # prepare for event log
+    python manage.py migrate eventlog
```

Comment:
I don't think you need a separate migrate for eventlog, the general migrate above should pick this up.

Reply:
indeed! removed the line.